packages/flutter/lib/src/material/material.dart

```diff
@@ -235,29 +241,59 @@ class Material extends StatefulWidget {
-  /// [ThemeData.applyElevationOverlayColor] is true and [ThemeData.brightness]
-  /// is [Brightness.dark] then a semi-transparent overlay color will be
-  /// composited on top of this color to indicate the elevation.
+  /// If [ThemeData.useMaterial3] is true then a new optional [surfaceTintColor]
+  /// overlay may be applied on top of this color to indicate elevation.
+  ///
+  /// If [ThemeData.useMaterial3] is false and [ThemeData.applyElevationOverlayColor]
+  /// is true and [ThemeData.brightness] is [Brightness.dark] then a
+  /// semi-transparent overlay color will be composited on top of this
+  /// color to indicate the elevation. This is no longer needed for Material
+  /// Design 3, which uses [surfaceTintColor].
   ///
   /// By default, the color is derived from the [type] of material.
   final Color? color;
 
   /// The color to paint the shadow below the material.
   ///
-  /// If null, [ThemeData.shadowColor] is used, which defaults to fully opaque black.
+  /// When [ThemeData.useMaterial3] is true, if this is null then no drop
+  /// shadow will be rendered for this material. If it is non-null then this
+  /// color will be render a drop shadow below the material.
   ///
-  /// Shadows can be difficult to see in a dark theme, so the elevation of a
-  /// surface should be portrayed with an "overlay" in addition to the shadow.
-  /// As the elevation of the component increases, the overlay increases in
-  /// opacity.
+  /// When [ThemeData.useMaterial3] is false, if this is null then
+  /// [ThemeData.shadowColor] is used, which defaults to fully opaque black.
   ///
   /// See also:
-  ///
+  ///  * [ThemeData.useMaterial3], which determines the default value for this
+  ///    property if it is null.
   ///  * [ThemeData.applyElevationOverlayColor], which turns elevation overlay
   /// on or off for dark themes.
   final Color? shadowColor;
 
+  /// The color of the surface tint overlay applied to the material color
+  /// to indicate elevation.
+  ///
+  /// Material Design 3 introduced a new way for some components to indicate
+  /// their elevation by using a surface tint color overlay on top of the
+  /// base material [color]. This overlay is painted with an opacity that is
+  /// related to the [elevation] of the material.
+  ///
+  /// If [ThemeData.useMaterial3] is false, then this property is not used.
+  ///
+  /// If [ThemeData.useMaterial3] is true and [surfaceTintColor] is not null,
+  /// then it will be used to overlay the base [color] with an opacity based
+  /// on the [elevation].
+  ///
+  /// Otherwise, no surface tint will be applied.
+  ///
+  /// See also:
+  ///
+  ///   * [ThemeData.useMaterial3], which turns this feature on.
+  ///   * [ElevationOverlay.applySurfaceTint], which is used to implement the
+  ///     tint.
+  ///   * https://m3.material.io/styles/color/the-color-system/color-roles
+  ///     which specifies how the overlay is applied.
+  final Color? surfaceTintColor;
```

Comment:
I think this should live in `ColorScheme`, with `Material.surfaceTintColor` defaulting to it, no?

Reply:
When we add it to color scheme, it still won't be the default for the `Material` widget. The default is null which will indicate that there should be no color overlay. For components that use the Material widget and need a surface tint overlay, they can use the new color in the color scheme for its default color.

Reply:
Gotcha